example/material.json

```diff
@@ -72,6 +72,11 @@
         {
             "name": "inchi_key",
             "value": ""
+        },
+        {
+            "name": "molecular_weight",
+            "value": 0,
```

Comment:
technically materials won't have a molecular weight, so I was unsure of what to put for a value here as `null` does not work with integer values.

Reply:
We can create "molecule.json" for both example and schema, where molecule is reusing most of the material content and has `isNonPeriodic` set to true